cypress/integration/vote_for_card_spec.js

```diff
@@ -0,0 +1,42 @@
+describe('Vote for a card', () => {
+    beforeEach(() => {
+        cy.login('unicorn');
+        /* Connect a second user using our fake client */
+        cy
+            .request(`http://localhost:12346/connect?url=${encodeURIComponent(Cypress.config().baseUrl)}`)
```

Comment:
maybe use a variable so that `http://localhost:12346` needn't be hardcoded and copy-pasted

Reply:
worth looking at the `URL` class to safely construct URLs programmatically (i.e. without relying on string concatenation). this may also provide a better alternative to URI-encoding query params on a case-by-case basis

Reply:
I tried switching it, but it didn't work straight away, so I'll create an issue and put more thought into it